src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -592,73 +594,99 @@ public String formatDateTime(GregorianCalendar time) {
 		return format.format(time.getTime());
 	}
 
-	private void ajustWidth() {
-		for (int i = 0; i < pColumwidth.length; i++) {
-			jTableInventory.getColumnModel().getColumn(i).setMinWidth(pColumwidth[i]);
-		}
-	}
-
 	private JComboBox<String> getComboBox() {
 		if (statusComboBox == null) {
 			statusComboBox = new JComboBox<String>();
```

Comment:
```suggestion
			statusComboBox = new JComboBox<>();
```
Replace explicit type object with <>